docs/content/opa-1.md

```diff
@@ -0,0 +1,201 @@
+This should be useful for cases when pre-OPA v1.0 functionality is desired but the users themselves have no control over the OPA deployment and hence cannot set the CLI flag.
+
+## Enforce use of `if` and `contains` keywords in rule head declarations
+
+Currently, there is semantic ambiguity between rules like `a.b {true}` and `a.b.c {true}`. Although syntactically similar, the former generates a set with the entry `b` at path `data.a`, while the latter generates an object with the attribute `"c": true` at path `data.a.b`. 
+This inconsistency makes it difficult for new users to understand how Rego works. 
+The `if` keyword is more than just syntactic sugar. When used in a rule head, that rule doesn't contribute to a partial set unless the `contains` keyword is also used. E.g. `a.b if {true}` will generate an object with the attribute `"b": true` at path `data.a`.
+To make things simpler in the future, OPA v1.0 will require the usage of `if` and `contains` keywords when declaring rules. This would mean:
+
+* All rules are single-value by default. When the value is omitted from the head, it defaults to `true`.
+* To make rules multi-value (i.e. partial set rules), use the `contains` keyword to convert the value into a set.
+
+The `contains` keyword is required to disambiguate rules that generate a single value from rules that generate multiple values. 
+The `if` keyword ensures that the semantics of rules does not change between today and v1.0. The table below illustrates why `if` is required.
+
+| rule              | output today             | output in v1.0           |
+|-------------------|--------------------------|--------------------------|
+| p { true }        | {"p": true}              | compile error            |
+| p.a { true }      | {"p": {"a"}}             | compile error            |
+| p.a.b { true }    | {"p": {"a": {"b": true}} | compile error            |
+| p if { true }     | {"p": true}              | {"p": true}              |
+| p.a if { true }   | {"p":{"a": true}}        | {"p":{"a": true}}        |
+| p.a.b if { true } | {"p": {"a": {"b": true}} | {"p": {"a": {"b": true}} |
+| p contains “a”    | {"p": {"a"}}             | {"p": {"a"}}             |
+
+If the Rego language was changed so that all rules were single-value by default, unless the `contains` keyword was used to make them multi-value, then the outcome of a rule like `p.a { true }` would change between today and v1.0 without generating an error.
+Generating errors in this case is preferable to changing the semantics of existing rules. Whereby, use of the `if` keyword will be a requirement in OPA v1.0, as this is also backwards compatible with older versions of OPA.
+
+In OPA v1.0, the `if` keyword is only required for rules with a declared body. Constants, rules that only consist of a value assignment, do not require `if`.
```

Comment:
I think we can make this clearer by saying, "so all of the following forms are unchanged" or similar.

Other option is to remove the `output in v1.0` col since it's the same?